extensions/all-extensions/pom.xml

```diff
@@ -23,6 +23,25 @@
             <artifactId>mule-module-file</artifactId>
             <version>${project.version}</version>
             <type>zip</type>
+            <exclusions>
```

Comment:
I don' think this is the right solution. As a developer, how would I know at design time that those dependencies won't be visible in runtime and I have to repackage them? These exclusions should be on the core poms